great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -500,7 +500,9 @@ def get_metrics(
             if attributed_resolved_metrics is None:
                 attributed_resolved_metrics = AttributedResolvedMetrics(
                     batch_ids=batch_ids,
-                    metric_attributes=metric_configuration.metric_value_kwargs,
+                    metric_attributes=Attributes(
```

Comment:
Probably something for another PR, but this import reads as nearly meaningless to me: `from great_expectations.types.attributes import Attributes`

Reply:
@NathanFarmer Not sure what the issue is -- happy to discuss (we do have names of modules and classes that read similarly).  Thanks.

Reply:
@alexsherstinsky we have stated as a team that we intend to remove `types` directories/modules as they are too generically named. The name `Attributes` is also a very generic concept. Reading this import tells me nothing about what this class is meant to represent.

Reply:
@NathanFarmer I see -- agreed with both (the `types` is clearly a remnant of the old structure).  Thanks.